tests/test_ilastik/test_workflows/test_HeadlessPixelClassificationWorkflow.py

```diff
@@ -224,3 +231,128 @@ def test_distributed_results_are_identical_to_single_process_results(
         distributed_50x50block_data = dataset[()]
 
     assert (single_process_out_data == distributed_50x50block_data).all()
+
+
+@pytest.fixture
+def ome_zarr_store_on_disc(tmp_path) -> Dict[str, numpy.ndarray]:
+    """Sets up a zarr store of a random image at raw scale and a downscale.
+    Returns a dictionary of the expected datasets (scale_key: numpy.ndarray)"""
+    zarr_dir = tmp_path / "some.zarr"
+    zarr_dir.mkdir(parents=True, exist_ok=True)
+
+    dataset_shape = [3, 100, 100]  # cyx - to match the 2d3c project
+    scaled_shape = [3, 50, 50]
+    chunk_size = [3, 64, 64]
```

Comment:
is it valid to have a chunk size bigger than, say the scaled shape?

Reply:
Yes, it just means the entire dataset is stored in one file